packages/alfa-act/src/diagnostic.ts

```diff
@@ -48,6 +48,8 @@ export namespace Diagnostic {
   export function isDiagnostic(value: unknown): value is Diagnostic {
     return value instanceof Diagnostic;
   }
+
+  export const defaultDiagnostic = Diagnostic.of("No extra information");
```

Comment:
```suggestion
  export const default = Diagnostic.of("No extra information");
```
It's already namespaced as `Diagnostic.default`, so we can keep a DRY name.

Reply:
Unfortunately, `default` seems to be reserved, would `defaultValue` be better?

Reply:
Ah yes 🙈 
`fallback`? `empty`?
`empty` sounds good and convey the idea: there is nothing to see here.